File: java/omnia/data/structure/mutable/MutableUndirectedGraph.java
package omnia.data.structure.mutable;

import omnia.data.structure.UndirectedGraph;

public interface MutableUndirectedGraph<E> extends MutableGraph<E>, UndirectedGraph<E> {

  void addEdge(E first, E second);

  boolean removeEdge(E first, E second);
}

File: src/omnia/data/structure/mutable/MutableUndirectedGraph.kt
package omnia.data.structure.mutable

import omnia.data.structure.UndirectedGraph

interface MutableUndirectedGraph<E> : MutableGraph<E>, UndirectedGraph<E> {
  fun addEdge(first: E, second: E)
  fun removeEdge(first: E, second: E): Boolean
}